### ISSUE TITLE
Suggestion - Furnace Minecarts Scale Speed with Fuel Type

### ISSUE THREAD

**Issue by StygianEmperor** [ndrp/faster-minecarts#3]
I was imagining a system by which, if you were able to load a minecart with fuels other than coal, they would increase its speed proportionally with the "quality" of the fuel (with some kind of cap obviously), rather than all of that going to the cart's duration. Maybe there would be some way to choose whether you got more speed or burn time, though I realize this idea is getting progressively more complex, and I for sure don't know how to pull it off.

Also, I know how often Fabric mods get Forge port requests, so apologies if this is irritating, but even without that feature this seems like exactly a mod I wanted to use; so I'd like to gently appeal for a Forge version. I won't press the issue though.

**Comment by faultycore** [comment]
Good idea, I'm actually working on different types of Minecarts right now, that you can craft with Gold or Diamonds that will have different top speeds. But the fuel idea is something I'll keep in mind.
